src/mcpadapt/core.py

```diff
@@ -135,10 +135,15 @@ class MCPAdapt:
     >>> # async usage with sse
     >>> async with MCPAdapt({"host": "127.0.0.1", "port": 8000}, SmolAgentAdapter()) as tools:
     >>>     print(tools)
+
+    You can also specify connect_timeout in seconds to the mcp server (default is 30s).
     """
 
     def __init__(
-        self, serverparams: StdioServerParameters | dict[str, Any], adapter: ToolAdapter
+        self,
+        serverparams: StdioServerParameters | dict[str, Any],
+        adapter: ToolAdapter,
+        connect_timeout=30,
     ):
```

Comment:
would be good to have a docstring to explain the connect_timeout parameter and then you could move `You can also specify connect_timeout in seconds to the mcp server (default is 30s).` directly in the docstring